src/lsm/grid.zig

```diff
@@ -292,6 +292,14 @@ pub fn GridType(comptime Storage: type) type {
             grid.assert_not_writing(address, block.*);
             grid.assert_not_reading(address, block.*);
 
+            if (constants.verify) {
+                for (&[_][]BlockPtr{ &grid.read_iop_blocks, grid.cache_blocks }) |blocks_used| {
```

Comment:
`assert_not_reading()` above should already checks `read_iops` for the relevant `read_iop_blocks` I assume (double check?)

Reply:
I think it only checks reads in flight. But maybe not worth checking the idle iops.